extensions/agroal/runtime/src/main/java/io/quarkus/agroal/runtime/DataSourceJdbcRuntimeConfig.java

```diff
@@ -111,17 +82,17 @@
      * no leaks are happening.
      */
     @ConfigItem(defaultValue = "true")
-    public boolean detectStatementLeaks;
+    public boolean detectStatementLeaks = true;
```

Comment:
@Sanne you want this to be default = true?

Reply:
It was already defaulted to true (see the ConfigItem) so if we need to discuss this, let's not block this PR.

Reply:
Sure,

Reply:
yes we want that to be true (as it currently is).  We'll have ORM bypass it, but that's for another day ;)